easybuild/easyconfigs/b/Boost/Boost-1.65.0-CrayGNU-17.08-cray-python2.eb

```diff
@@ -0,0 +1,28 @@
+# contributed by Luca Marsella (CSCS)
+name = 'Boost'
+version = '1.65.0'
+versionsuffix = '-cray-python2'
```

Comment:
What do you think of `-python2` only as versionsuffix? The toolchain `CrayGNU` already suggests the use of `cray-python` and the version `17.08` will hint at the use of version `17.06.1`.